web/src/App.svelte

```diff
@@ -1,12 +1,21 @@
 <script lang="ts">
+  // @ts-nocheck
+  import { Auth0Context } from '@dopry/svelte-auth0'
   import { Router } from '@roxi/routify'
   import Window from '$/components/Window.svelte'
   import { routes } from '../.routify/routes'
 </script>
 
-<Router {routes} />
+<Auth0Context
+  domain="eseuri-prod.eu.auth0.com"
+  client_id="NJgHdaXoixiNUpEks7b3TtYi95ZH433P"
+  audience="https://eseuri-prod.eu.auth0.com/api/v2/"
```

Comment:
Astea nu ar trebui sa existe in repository, vezi [cum am facut eu](https://github.com/julien2429/Eseuri_fork/commit/37ee2d75089694c644d77789cd5f876e046d7b9e). Desi acolo au ramas doua dintre ele, nu ar trebui sa existe niciuna dintre datele astea, de fapt.
Citeste si [documentatia Vite](https://vitejs.dev/guide/env-and-mode.html#env-variables).